ssc/cmod_pvsamv1.cpp

```diff
@@ -2711,11 +2772,12 @@ void cm_pvsamv1::exec( ) throw (compute_module::general_error)
 	*********************************************************************************************** */
 	idx = 0; ireport = 0; ireplast = 0; percent_baseline = percent_complete;
 	double annual_battery_loss = 0;
+	wdprov->rewind();
```

Comment:
Why did reading the weather file need to be added here? Seems like there must be an existing variable containing weather file temperature here...

Reply:
The inverter's CalculateACPower function requires wf. tdry, and the previous instance of weather file was inside the previous for loop (DC) and so is not available in this loop (AC)

Reply:
What about using the tdry output variable here?

Hopefully something that will be addressed with @nickdiorio 's IOManager. :)